common/src/test/scala/uk/ac/wellcome/test/utils/SQSLocal.scala

```diff
@@ -16,12 +24,27 @@ trait SQSLocal extends Suite with BeforeAndAfterEach {
       new EndpointConfiguration(s"http://localhost:9324", "localhost"))
     .build()
 
-  val idMinterQueue = "es_id_minter_queue"
-  val idMinterQueueUrl = sqsClient.createQueue(idMinterQueue).getQueueUrl
+  def queueName: String
+
+  // Use eventually to allow some time for the local SQS to start up.
+  // If it is not started all suites using this will crash at start up time.
+  // Did not happen locally but seems to be needed on CI
```

Comment:
I don't think this is really related to Travis, it could affect any CI and in fact even a slower local development machine